android-client-sdk/proguard-rules.pro

```diff
@@ -18,4 +18,6 @@
 
 # If you keep the line number information, uncomment this to
 # hide the original source file name.
-#-renamesourcefileattribute SourceFile
\ No newline at end of file
+#-renamesourcefileattribute SourceFile
+-keep class kotlin.Metadata { *; }
+-keep class com.devcycle.sdk.android.model.** { *; }
```

Comment:
also lets make this part of the consumer proguard files so it ships with the sdk now

Reply:
oh yeah i forgot that